Source/Utils/WarpXUtil.cpp

```diff
@@ -385,7 +385,7 @@ parseStringtoReal(std::string str)
 int
 parseStringtoInt(std::string str, std::string name)
 {
-    amrex::Real rval = parseStringtoReal(str);
+    amrex::Real rval = static_cast<amrex::Real>(parseStringtoReal(str));
```

Comment:
`const`-ify + avoiding type repetition from the RHS:
```suggestion
    auto const rval = static_cast<amrex::Real>(parseStringtoReal(str));
```